src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -346,6 +352,15 @@ case class Access(e: Accessible, p: Expr)(val info: Source.Parser.Info) extends
   require(p.typ.isInstanceOf[PermissionT], s"expected an expression of permission type but got $p.typ")
 }
 
+sealed trait TerminationMeasure extends Assertion
```

Comment:
Could be simplified similarly to `PTerminationMeasure`. Again, for time constraints, we can keep the current class structure.